src/components/volunteer/VolunteerFormGroup.tsx

```diff
@@ -138,21 +150,66 @@ export const VolunteerFormGroup = ({
         <Controller
           name="visibility"
           render={({ onChange, value }) => (
-            <CheckBox
-              checked={!!value}
-              onPress={() => onChange(!value)}
-              title="Öffentlich"
-              checkedColor={colors.accent}
-              checkedIcon="check-square-o"
-              uncheckedColor={colors.darkText}
-              uncheckedIcon="square-o"
-              containerStyle={styles.checkboxContainerStyle}
-              textStyle={styles.checkboxTextStyle}
-            />
+            <>
+              <Input
+                hidden
+                name="Sichtbarkeit"
+                label={texts.volunteer.visibility}
+                placeholder={texts.volunteer.visibility}
+                control={control}
+              />
```

Comment:
`name` should match the form field, `visibility` in this case.
but you do not need the input at all. it is already handled by the `Controller` with `name="visibility"`

Reply:
for rendering a label, we use `<Label>{...}</Label>`